modules/seqexec/server/src/main/resources/Gmos.xml

```diff
@@ -469,10 +469,20 @@
             <type>STRING</type>
             <description>GMOS detector controller s/w</description>
         </attribute>
+        <attribute name="obsPrep">
+            <channel>sad:dc:prep.VAL</channel>
+            <type>INTEGER</type>
+            <description>Observation is preparing</description>
+        </attribute>
         <attribute name="obsAcq">
             <channel>sad:dc:acq.VAL</channel>
             <type>INTEGER</type>
-            <description>Observation is aquiring</description>
+            <description>Observation is acquiring</description>
```

Comment:
Why is it "Acquiring observation" for F2 but "Observation is acquiring" for Gmos? Should these be consistent?

Reply:
(Same with others: Reading out observation vs. Observation is reading out and Preparing observation vs. Observation is preparing?)

Reply:
That's boring :)